book/website/community-handbook/newsletters/newsletters-style.md

```diff
@@ -0,0 +1,49 @@
+    - Create link for email ids using this Markdown syntax - ``[real-email-id](mailto:real-email-id)``
+- **Quoting others**: Use smaller than (>) symbol followed by a space before the quoted sentence. For example:
+    `> This is my legendary quote.` will appear as:
+    > This is my legendary quote.
+- **Header and styling**: The newsletter title is the top header.
+    - Different sections as suggested in the newsletters are second-level headers and the sub-sections are third-level headers.
+    - Use bold letters, italics, hyperlinked texts and quotations wherever applicable
+    - The project name, _The Turing Way_, should be italicised.
+    - Use line breaks for each line consistent with _The Turing Way_ writing format.
+    - Leave at least one line space after each section and subsection.
+- **Language and tone**: Keep the overall language simple and jargon-free, see [_The Turing Way_ style guide](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md#style-guide) for reference.
+    - The tone should be welcoming, friendly and preferably informal.
+    This can be personal to the author's writing style.
+    - Ask more than one person to review your draft to make sure that its content is easy to understand and written clearly.
+    - If using content from a language or culture different from your own, ask people with that language or culture to review your draft to make sure that contents are not misrepresented.
+- **Use of emojis**: It is encouraged to use emoji (*show your personality*) 😇, but keep it simple, neutral and positive.
+    - Be aware that that ambiguous emojis can be misinterpreted by different readers.
+    - When in doubt, ask someone to review your draft.
+- **Use of images**: Only use relevant images linked to the news item in the newsletter.
+    - Make sure that the images are available under CC-BY license or approved to be reused by the owners.
+    - Avoid using memes, images with political or sexual innuendo, or anything that is not directly related to the community.
+    - When drafting the newsletter in a HackMD, drag-n-drop an image into the editor or copy-paste an image to automatically upload the image to [imgur](https://en.wikipedia.org/wiki/Imgur).
+    - When drafting the newsletter on the GitHub, upload the images in the folder "the-turing-way/communications/newsletters/".
+    - File naming convention for the images is "short-name-monthYYYY.png", where the short-name should be replaced with the identifiable short name of the image, the month should be replaced by the short name of the month and YYYY should be replaced by the year.
+    - File extension can be '.jpg', '.png' or other with compatible image file type.
+    - Use Markdown syntax to link them in the newsletter. Example syntax, `![Alt: Description of the image - this is not the title but actual explanation of the image](image/path)`
```

Comment:
Not sure if this should be two lines. I think this could be 2 points, one explaining how to link and one highlighting alt text.

Reply:
Adding cross ref to the style guide.